objects.go

```diff
@@ -579,3 +579,78 @@ func (element *Element) parse(line string) error {
 	}
 	return nil
 }
+
+// Object implementation for Flowtable
+func (flowtable *Flowtable) validate(verb verb) error {
+	switch verb {
+	case addVerb, createVerb:
+		if flowtable.Name == "" {
+			return fmt.Errorf("no name specified for flowtable")
+		}
```

Comment:
The man page implies that `Priority` and at least one `Devices` are required as well (though the man page may be wrong).

Reply:
I tested all these commands locally
```
nft add flowtable ip kube-proxy kube-proxy-flowtable {}
root@kind-control-plane:/# nft list table kube-proxy
table ip kube-proxy {
        flowtable kube-proxy-flowtable {
                hook ingress priority filter
        }
}
```
empty seems to apply the defaults

without the square brackets it fails

```
ft add  flowtable ip kube-proxy kube-proxy-flowtable
Error: syntax error, unexpected newline, expecting '{'
add flowtable ip kube-proxy kube-proxy-flowtable
```